lnbits/core/views/api.py

```diff
@@ -181,11 +181,11 @@ async def api_payments_create_invoice(data: CreateInvoiceData, wallet: Wallet):
 
     lnurl_response: Union[None, bool, str] = None
     if data.lnurl_callback:
-        if "lnurl_balance_check" in data:
+        if hasattr(data, "lnurl_balance_check"):
```

Comment:
This is not the same thing. 
<img width="397" alt="image" src="https://user-images.githubusercontent.com/93376500/181191143-bad071a6-e3c4-4099-9b19-7c3cd3d9814b.png">

Reply:
reverted